addons/website_hr/controllers/main.py

```diff
@@ -1,19 +1,16 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp import http
-from openerp.http import request
+from odoo import http
+from odoo.http import request
 
-class website_hr(http.Controller):
+class WebsiteHr(http.Controller):
 
     @http.route(['/page/website.aboutus', '/page/aboutus'], type='http', auth="public", website=True)
     def blog(self, **post):
-        hr_obj = request.registry['hr.employee']
-        if request.registry['res.users'].has_group(request.cr, request.uid, 'base.group_website_publisher'):
-            employee_ids = hr_obj.search(request.cr, request.uid, [], context=request.context)
+        HrEmployee = request.env['hr.employee']
+        if request.env['res.users'].has_group('base.group_website_publisher'):
+            employees = HrEmployee.search([])
         else:
-            employee_ids = hr_obj.search(request.cr, request.uid, [('website_published', '=', True)], context=request.context)
-        values = {
-            'employee_ids': hr_obj.browse(request.cr, request.uid, employee_ids,
-                                          request.context),
-        }
-        return request.website.render("website.aboutus", values)
+            employees = HrEmployee.search([('website_published', '=', True)])
+        return request.website.render("website.aboutus", {'employee_ids': employees})
```

Comment:
can you rename key: employees instead of employee_ids which passed in render template ?